tools/release/backport_pr.sh

```diff
@@ -24,59 +24,104 @@ ensure_command () {
   fi
 }
 
+display_usage () {
+  cat << EOF >/dev/stderr
+USAGE: $0 PR_ID GITHUB_USER BACKPORT_BRANCHES REVIEWERS [-c PER_BACKPORT_COMMAND]
```

Comment:
For consistency, replace PR_ID with PR_NUMBER throughout. The variable used in the script is $PR_NUMBER. Or alternatively, change the variable to $PR_ID.